ls1intum/Artemis
# Allow editors to edit lectures and units

### Checklist

### Motivation and Context


**Editors** should be able to **edit** lectures and their corresponding units.

### Steps for Testing


1. Log in as Editor, check if you can to the following:
2. Update, create, delete lectures
3. Update, create, delete, reorder lecture units and attatchments
4. Verify that Instructors/Admins can still do the same
